### PR TITLE
Dev nearby connection button

### PR DESCRIPTION
**Pull request by L-roro**
Small pull request from what we were working on. Not to main.

### CODE REVIEW

File: app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/profile/UserProfile.kt [hunk 1]
```diff
@@ -62,17 +61,20 @@ fun UserProfile(
...
+                navController.navigate(Screen.ReceiveNearbyWork.route)
+            }
+            Button(Constant.USER_PROFILE_SETTINGS)
         }
     }
 }
 
 @Composable
-private fun Button(text: String){
+private fun Button(text: String, onClick: () -> Unit = {}) {
```
Comment (Lynoxe) [inline app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/profile/UserProfile.kt:75]: It's maybe a bit confusing to have it named `Button`, giving a more insightful name to it could be useful

File: app/src/main/AndroidManifest.xml
Comment (Lynoxe) [inline app/src/main/AndroidManifest.xml:1]: Why was it needed ? (removing `android:minSdkVersion="29"`)

**Review by Lynoxe — APPROVED** [review]
The CI doesn't pass, is it expected?..

File: app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/profile/UserProfile.kt (hunk 1 above)
Comment (L-roro) [reply app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/profile/UserProfile.kt:75]: I actually don't fully understand what the purpose of that function was, but I hesitated to remove it. At the end of the day, it would be almost identical to just use the jetpack compose button.

File: app/src/main/AndroidManifest.xml
Comment (L-roro) [reply app/src/main/AndroidManifest.xml:1]: I actually don't think it was needed. The problem is that we had multiple issues because all the sdk versions were changed and were given min and max values, when there previously were none. Doing so incurred multiple problems, specially because they were poorly fixed. When we found this problem, I reverted it to the original version without thinking too much, while the other branch didn't revert this change in particular.
TLDR: could be removed from the pr

**Comment by L-roro** [comment]
Yeah, it does not pass the CI. I branched from Léo's branch and didn't realise that it didn't. There were compilation errors on the tests that I more or less solved, but unfortunately, there are more problems. It looks like the dependencies on the gradle files were updated and testing is now broken as a result. I have spent a decent amount of time toying with the dependencies trying to solve this, without much success. I will try again tomorrow and Léo said the same.
